src/sentry/static/sentry/app/utils/tokenizeSearch.tsx

```diff
@@ -379,3 +330,57 @@ function formatTag(tag: string) {
 function formatQuery(query: string) {
   return query.replace(/^["\(]+|["\)]+$/g, '');
 }
+
+export function updateQuery(
```

Comment:
After reading @evanh's comment above, it seems to me that the intent of the `QueryResults` class is to provide the basic operations in order to manipulate the query. As such, I don't think that every `QueryResults` should be aware of cell actions. 

n fact, now that I think about it, it might make more sense to move this `updateQuery` into/closer to `CellActions` as this is an operation that can only be done when cell actions are possible.

Reply:
Fair points. When I see function that takes an object and some primitives and then does a bunch of operations on the object I feel compelled to ask :smile:

Reply:
This is now in `cellActions.tsx`